### PR TITLE
docs: add icon guide

### PR DESCRIPTION
**Pull request by georgexu99**
addresses: https://github.com/electron-userland/electron-forge/issues/2922

### CODE REVIEW

**Comment by malept** [comment]
This is a good start!

The guide needs to mention that installers also usually have icons and the ways to configure those.

File: guides/create-and-add-icons.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+config: {
+    forge: {
+      packagerConfig: {
+        icon: "/path/to/icon"
+      },
+    }
+}
+```
+
+`electron-forge` will automatically add the correct extension for each platform, thus appending `.ico` or `.icns` to the path is not required.
```
Comment (VerteDinde) [inline guides/create-and-add-icons.md:51]: This is great, and very helpful - lots of questions around this specifically.

**Review by VerteDinde — APPROVED** [review]
This looks excellent, thanks for taking this on!

One thing we might think about adding is best practices around icon sizing, because I know that's another question that often comes up. I don't think we need to hold this PR for that though, we can merge as is 🙂

**Review by VerteDinde — APPROVED** [review]

File: guides/create-and-add-icons.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+// In package.json, or forge.config.js if you have set config.forge to a JavaScript file path in package.json
+config: {
+    forge: {
+        makers: [
+            {
+                name: "@electron-forge/maker-squirrel",
+                config: {
+                    options: {
+                        // An URL to an ICO file to use as the application icon (displayed in Control Panel > Programs and Features).
+                        iconurl: "https://url/to/icon.ico",
```
Comment (malept) [inline guides/create-and-add-icons.md:111]: issue: I think it's camel cased somehow, like iconUrl or iconURL.

**Pull request merged**